taprpc/marshal.go

```diff
@@ -474,6 +475,13 @@ func MarshalAsset(ctx context.Context, a *asset.Asset,
 		IsBurn:                 a.IsBurn(),
 	}
 
+	decDisplay.WhenSome(func(u uint32) {
+		rpcAsset.DecimalDisplay = &DecimalDisplayOpt{
```

Comment:
See my previous comment. If we want the boolean `IsSet` to be useful, we should set `DecimalDisplay: &DecimalDisplayOpt{},` above where we initialize `rpcAsset`.

Reply:
could add this, to make it more explicit

Reply:
ok so I went with the approach where `is_set` is omitted, and we rely on the nested message being nil or not to signal whether a value has been set